kaggle/kaggle_bigquery.R

```diff
@@ -27,8 +27,15 @@ TokenBigQueryKernel <- R6::R6Class("TokenBigQueryKernel", inherit = Token2.0, li
     if (KAGGLE_USER_SECRETS_TOKEN == '') {
       stop("Expected KAGGLE_USER_SECRETS_TOKEN environment variable to be present.", call. = FALSE)
     }
-    request_body <- list(JWE = KAGGLE_USER_SECRETS_TOKEN, Target = 1)
-    response <- POST(paste0(KAGGLE_BASE_URL, GET_USER_SECRET_ENDPONT), body = request_body, encode = "json")
+    request_body <- list(Target = 1)
+    auth_header <- paste0("Bearer ", KAGGLE_USER_SECRETS_TOKEN)
+    headers <- add_headers(c("X-Kaggle-Authorization" = auth_header))
+    response <- POST(paste0(KAGGLE_BASE_URL, GET_USER_SECRET_ENDPONT),

```

Comment:
nit: ENDPOINT (missing a I), which is also misspelled on line 13

Reply:
Done